spec/exporters/publishing_api_publisher_spec.rb

```diff
@@ -66,7 +66,7 @@
       it "asks the publishing api to publish the section with the specific update_type" do
         subject.call
 
-        expect(publishing_api).to have_received(:publish).with(section_id, "republish")
+        expect(publishing_api).to have_received(:publish).with(section_uuid, "republish")
       end
     end
   end
```

Comment:
Minor: I think it would make things clearer if you distinguish this commit from the other 2 by appending "in `PublishingAPIPublisher`" to the subject line. Ditto for the other three commits.